backend-project/small_eod/cases/models.py

```diff
@@ -18,20 +18,29 @@ def with_counter(self):
 class Case(TimestampUserLogModel):
     objects = CaseQuerySet.as_manager()
 
-    name = models.CharField(max_length=256)
-    comment = models.CharField(max_length=256)
-
-    tag = models.ManyToManyField(to=Tag, blank=True)
-    feature = models.ManyToManyField(to=Feature, blank=True)
+    name = models.CharField(max_length=256, verbose_name=_("Name"),
+                            help_text=_("Case's name. Name cannot be longer than 256 characters.")
```

Comment:
Czy długość nie jest już teraz udokumentowana w modelu w jednoznacznej, maszynowej postaci?

Reply:
Jak mogę teraz dostać się do tego PR, żeby to poprawić?

Reply:
Musisz pushować do brancha ```#122-verbose_name_help_text_cases``` w Twoim forku ( https://github.com/efefre/small_eod/tree/%23122-verbose_name_help_text_cases ). Pull request sam się zaktualizuje po zmianach na branchu źródłowym.

Reply:
Ehhh... nie mogłam się przełączyć wczoraj na brancha lokalnie, więc myślałam, że jest jakaś inna procedura. Ale dzisiaj już doszłam co jest problemem. Prawdopodobnie # w nazwie brancha. Przez terminal komenda checkout nie pozwalała mi na przejście na tę gałąź. Dzisiaj ściągnęłam SourceTree i tam przełączyłam się bez problemu.